src/main/java/org/hyperledger/fabric_ca/sdk/HFCAClient.java

```diff
@@ -1199,6 +1200,106 @@ public Enrollment idemixEnroll(Enrollment enrollment, String mspID) throws Enrol
         }
     }
 
+    /**
+     * idemixEnrollSerializedStr returns an Identity Mixer Enrollment As a Serialized String, which supports anonymity and unlinkability
```

Comment:
It's not returning an Idemix enrollment as a serialized string. It's returning an Enrollment interface, which provides only these methods:

```java
public interface Enrollment {
    PrivateKey getKey();
    String getCert();
}
```
In the returned implementation, both of those methods return `null` so it is not a useful return value.